pyqt-apps/siriushla/li_va_control/details.py

```diff
@@ -0,0 +1,324 @@
+""" All the detail Chart Windows """
+
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import QWidget, QGroupBox, QVBoxLayout, QHBoxLayout, QLabel
+from ..widgets import SiriusMainWindow
+from .functions import BaseFunctionsInterface
+from .util import IPS_DETAILS, PVS_CONFIG, VGC_DETAILS, COLORS
+
+class IpsDetailWindow(SiriusMainWindow, BaseFunctionsInterface):
+    """ Display IPS Detail Window"""
+
+    def __init__(self, parent=None, prefix='', id_num=''):
+        """Init."""
+        super().__init__(parent)
+        self.prefix = prefix
+        self.main_dev = PVS_CONFIG["Pump"]["prefix"]
+        self.devpref = self.prefix + self.main_dev + self.buildIdName(id_num)
+        self.setObjectName('LIApp')
+        title = "IPS "+str(id_num)+" Details"
+        self.setWindowTitle(title)
+        self._setupUi(title)
+
+    def buildIPSDetail(self, info):
+        """ Display IPS measurement complete information """
+        wid = QGroupBox()
+        lay = QHBoxLayout()
+        lay.setContentsMargins(0, 0, 0, 0)
+        wid.setLayout(lay)
+        for info_type, type_config in info.items():
```

Comment:
you can use `for type_config in info.values():` here, once you are not using the variable info_type in the loop